src/main/java/uk/gov/ons/entities/InputAddress.java

```diff
@@ -32,16 +40,44 @@
 	@CsvBindByName
 	private String longitude;
 	@CsvBindByName
+	@NotBlank(message = "UPRN cannot be blank")
 	private String uprn;
 	@CsvBindByName(column = "abp_code")
 	private String abpCode;
 	@CsvBindByName(column = "organisation_name")
 	private String organisationName;
 	@CsvBindByName(column = "estab_uprn")
 	private String estabUprn;
-
-	public String getAddressAll() {
 	
+	@JsonCreator
+	public InputAddress(@JsonProperty(value = "addressLine1", required = true) String addressLine1, 
```

Comment:
This is fine, but just for info, I think you might be able to add the @AllArgsContructor on top of class declaration.
and the @JsonProperty at the field level

Reply:
As discussed - the problem is that the ```@JsonCreator``` is required on the all args constructor.